go/vt/vttablet/tabletserver/query_executor.go

```diff
@@ -172,6 +171,10 @@ func (qre *QueryExecutor) Execute() (reply *sqltypes.Result, err error) {
 
 	switch qre.plan.PlanID {
 	case p.PlanSelect, p.PlanSelectImpossible, p.PlanShow:
+		if err := qre.tsv.txThrottler.Throttle(qre.plan.Plan, qre.options); err != nil {
+			return nil, vterrors.Errorf(vtrpcpb.Code_RESOURCE_EXHAUSTED, "Query throttled, cause: %v", err)
+		}
```

Comment:
How much logic is involved in this call to `qre.tsv.txThrottler.Throttle()` assuming I haven't configured anything related to the tx throttler? Trying to understand the impact to any given `SELECT` query.